dbt/models/location/schema.yml

```diff
@@ -271,7 +271,7 @@ unit_tests:
     model: location.crosswalk_year_fill
     given:
       - input: source("spatial", "parcel")
-        rows: &crosswalk-location-fill-unit-test-pin-and-year
+        rows: &location-crosswalk-fill-year-unit-test-pin-and-year
```

Comment:
Slightly clearer anchor naming than what I implemented in https://github.com/ccao-data/data-architecture/pull/719.